spinnaker-monitoring-daemon/spinnaker-monitoring/spectator_metric_transformer.py

```diff
@@ -329,13 +329,15 @@ def __prepare_transformation(self, transformation):
       if compare:
         transformation['_xform_func'] = lambda value: {to_tag: value == compare}
       else:
-        transformation['_xform_func'] = lambda value: {to_tag: value == 'true'}
+        transformation['_xform_func'] = lambda value: {
+            to_tag: value == 'true' or value == ''
```

Comment:
Is it intentional that empty string results in true?

Reply:
Nevermind; I just read the comment above.  Looks good.

Reply:
I agree it is scary [unusual] but it does seem to be appropriate for this use case. If it turns out not to hold for some cases, then they would need to explicitly specify the true condition in the transformation specification for that meter.